### ISSUE TITLE
CalledFromWrongThreadException

### ISSUE THREAD

**Issue by stefan-niedermann** [stefan-niedermann/nextcloud-deck#20]
```
2018-12-09 14:10:34.280 17670-20631/? E/AndroidRuntime: FATAL EXCEPTION: Thread-36
    Process: it.niedermann.nextcloud.deck, PID: 17670
    android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:7753)
        at android.view.ViewRootImpl.requestLayout(ViewRootImpl.java:1225)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.support.v4.widget.DrawerLayout.requestLayout(DrawerLayout.java:1303)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.support.constraint.ConstraintLayout.requestLayout(ConstraintLayout.java:3172)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.view.View.requestLayout(View.java:23093)
        at android.support.v7.widget.RecyclerView.requestLayout(RecyclerView.java:4202)
        at android.support.v7.widget.RecyclerView$RecyclerViewDataObserver.onChanged(RecyclerView.java:5286)
        at android.support.v7.widget.RecyclerView$AdapterDataObservable.notifyChanged(RecyclerView.java:11997)
        at android.support.v7.widget.RecyclerView$Adapter.notifyDataSetChanged(RecyclerView.java:7070)
        at it.niedermann.nextcloud.deck.ui.card.CardAdapter.setCardList(CardAdapter.java:79)
        at it.niedermann.nextcloud.deck.ui.stack.StackFragment$1.onResponse(StackFragment.java:81)
        at it.niedermann.nextcloud.deck.ui.stack.StackFragment$1.onResponse(StackFragment.java:75)
        at it.niedermann.nextcloud.deck.persistence.sync.adapters.DataBaseAdapter.lambda$respond$0(DataBaseAdapter.java:38)
        at it.niedermann.nextcloud.deck.persistence.sync.adapters.-$$Lambda$DataBaseAdapter$U3N7eFNxmQsJezBFG2B6ELKH524.run(Unknown Source:4)
        at java.lang.Thread.run(Thread.java:764)
```

**Comment by desperateCoder** [comment]
Oops. Didn't know a '#20' in commit-message would close this...But yeah. See the commit.

**Comment by AndyScherzinger** [comment]
@desperateCoder the commit message shouldn't close it put the merge of the commit to master does. Same for posting

Resolves #20 within the issue's description upon merge :) Gotta love automation 🚀
